CC/Sounder/Radio.cc

```diff
@@ -6,33 +6,55 @@
 void Radio::dev_init(Config* _cfg, int ch, double rxgain, double txgain)
 {
     // these params are sufficient to set before DC offset and IQ imbalance calibration
-    dev->setAntenna(SOAPY_SDR_RX, ch, "TRX");
-    dev->setBandwidth(SOAPY_SDR_RX, ch, _cfg->bwFilter);
-    dev->setBandwidth(SOAPY_SDR_TX, ch, _cfg->bwFilter);
+    if (!kUseUHD) {
+        dev->setAntenna(SOAPY_SDR_RX, ch, "TRX");
+        dev->setBandwidth(SOAPY_SDR_RX, ch, _cfg->bwFilter);
+        dev->setBandwidth(SOAPY_SDR_TX, ch, _cfg->bwFilter);
+        dev->setFrequency(SOAPY_SDR_RX, ch, "BB", _cfg->nco);
+        dev->setFrequency(SOAPY_SDR_TX, ch, "BB", _cfg->nco);
+    } else {
+        std::cout << "Init USRP channel: " << ch << std::endl;
+        dev->setAntenna(SOAPY_SDR_TX, ch, "TX/RX");
+        dev->setAntenna(SOAPY_SDR_RX, ch, "RX2"); // or "TX/RX"
+        dev->setFrequency(SOAPY_SDR_RX, ch, "BB", 0);
+        dev->setFrequency(SOAPY_SDR_TX, ch, "BB", 0);
+    }
 
     dev->setFrequency(SOAPY_SDR_RX, ch, "RF", _cfg->radioRfFreq);
-    dev->setFrequency(SOAPY_SDR_RX, ch, "BB", _cfg->nco);
     dev->setFrequency(SOAPY_SDR_TX, ch, "RF", _cfg->radioRfFreq);
-    dev->setFrequency(SOAPY_SDR_TX, ch, "BB", _cfg->nco);
+
+    std::cout << "Tx RF freq: " << dev->getFrequency(SOAPY_SDR_TX, ch, "RF")
+              << ", Rx RF freq: " << dev->getFrequency(SOAPY_SDR_RX, ch, "RF") << std::endl;
+    std::cout << "Tx BB freq: " << dev->getFrequency(SOAPY_SDR_TX, ch, "BB")
+              << ", Rx BB freq: " << dev->getFrequency(SOAPY_SDR_RX, ch, "BB") << std::endl;
```

Comment:
Sorry, I realized these print wouldn't be pretty at runtime with a lot of BS antennas. Perhaps we could comment it out or just use it in UHD mode!

Reply:
Done.